qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -81,6 +81,7 @@ def level_1_pass_manager(pass_manager_config: PassManagerConfig) -> StagedPassMa
     unitary_synthesis_plugin_config = pass_manager_config.unitary_synthesis_plugin_config
     timing_constraints = pass_manager_config.timing_constraints or TimingConstraints()
     target = pass_manager_config.target
+    call_limit = pass_manager_config.call_limit
```

Comment:
Are you adding this to the passmanager config somewhere? I don't see that change anywhere